src/components/grid/GridComponent.vue

```diff
@@ -198,7 +198,7 @@ export default Vue.extend({
   },
   filters: {
     formatCount: function (value) {
-      if (value === -1) {
+      if (value < 0) {
         return '1-15'
       } else if (isNaN(value)) {
```

Comment:
why is the NaN check needed ?

Reply:
I haven't checked why, but we can discuss about this function anyway, because I don't like the hardcoded 1-15.

Reply:
we found out that it's to structure the grid